Gemfile.lock

```diff
@@ -172,6 +172,8 @@ GEM
     kramdown (1.13.2)
     liquid (4.0.0)
     listen (3.0.6)
+      rb-fsevent (>= 0.9.3)
+      rb-inotify (>= 0.9.7)
```

Comment:
Are you using any Ruby?  Since it's not Jekyll, can `Gemfile.lock` be removed entirely?

Reply:
Yeah, so as I am not familiar with the Ruby environment, I am not using Ruby. If you say I can remove Gemfile.lock.

Reply:
Yes, since you're not using Ruby, please remove this.  Thanks!